app/models/rapporteringsperiode.server.ts

```diff
@@ -98,9 +98,10 @@ export async function hentPeriode(
   request: Request,
   periodeId: string,
   hentOriginal: boolean = true
-): Promise<IRapporteringsperiode> {
+): Promise<{ periode: IRapporteringsperiode }> {
   const url = `${DP_RAPPORTERING_URL}/rapporteringsperiode/${periodeId}`;
 
+  // TODO: Legg til x_corralationId som header
```

Comment:
FYI: dp-rapportering prøver å finne correlationId i XRequestId

Reply:
Stemmer! Hvis en request til dp-rapportering har "X-Request-ID: corr-id-123" i header vil den settes som korrelasjonsId. Responsen fra dp-rapportering vil også inneholde den samme headeren.

Reply:
Se [ca3350d](https://github.com/navikt/dp-rapportering-frontend/pull/242/commits/ca3350d7fb7e1ab104f9e82a5a0991e73773f761) for forslag. Der gjenbrukes corralationId hvis den finnes på requesten, er det riktig? Eller bør jeg alltid lage en ny for hvert kall frontenden gjør?